packages/diagrams/frontend/sirius-components-diagrams/src/renderer/DiagramRenderer.tsx

```diff
@@ -30,6 +30,7 @@ import {
   useReactFlow,
   useStoreApi,
 } from '@xyflow/react';
+import '@xyflow/react/dist/style.css';
```

Comment:
I'll have to be more careful because this should not change position, it will create bugs otherwise